### ISSUE TITLE
Missing count

### ISSUE THREAD

**Issue by wfalkwallace** [kpdecker/jsdiff#69]
`diff.diffLines('var a = 0;', 'var a = 0;\nvar b = 1;\nvar c = 3')` returns
```
[{
  added: undefined,
  count: 1,
  removed: true,
  value: "var a = 0;",
},
{
  added: true,
  count: 3,
  removed: undefined,
  value: "var a = 0;↵var b = 1;↵var c = 3",
}]
```

`diff.diffLines('', 'var a = 0;\nvar b = 1;')` returns
```
[{
  added: true,
  value: "var a = 0;↵var b = 1;",
}]
```

what happened to count?

**Comment by kpdecker** [comment]
There are some short-circuit edge cases [here](https://github.com/kpdecker/jsdiff/blob/master/src/diff/base.js#L23-L31) that don't run the tokenizer. `count` was intended more as an internal field, but it wouldn't be that much overhead to run the tokenizer for these special cases and include the field for those as well.

**Comment by kpdecker** [comment]
Released in 2.1.0
